lib/services/presets.go

```diff
@@ -521,6 +521,7 @@ func defaultAllowRules() map[string][]types.Rule {
 			types.NewRule(types.KindInstance, RO()),
 			types.NewRule(types.KindAssistant, append(RW(), types.VerbUse)),
 			types.NewRule(types.KindNode, RW()),
+			types.NewRule(types.KindAccessList, RW()),
```

Comment:
Is there any way we can add a test to prevent this from happening again? Despite the comment at the top of this function we still missed it.

Reply:
In a separate PR, I've modified our tests to hopefully cover this: https://github.com/gravitational/teleport/pull/32128 I made this a draft because I wanted your eyes on it first. At a glance it looks pretty safe, but I want to make sure there's no unintended behavior here.